apps/web/src/server/applications/case/s51/applications-s51.validators.js

```diff
@@ -26,8 +26,11 @@ export const s51ValidatorsDispatcher = async (request, response, next) => {
 		'advice-details': validateS51Details('advice', 'advice given')
 	};
 
-	if (validators[step]) {
-		return validators[step](request, response, next);
+	const validatorKey = Object.entries(validators).find((entry) => entry[0] === step);
```

Comment:
This also looks a bit obscure, would the following suggested change pass the scan?:
```
	const validator = validators[step];
	if (typeof validator === 'function') {
		return validator(request, response, next);
	}
```